src/suites/cts/validation/validation_test.ts

```diff
@@ -0,0 +1,35 @@
+import { GPUTest } from '../gpu_test.js';
+
+export class ValidationTest extends GPUTest {
+  async init(): Promise<void> {
+    await super.init();
+
+    this.device.pushErrorScope('validation');
+  }
+
+  async finalize(): Promise<void> {
```

Comment:
Can you move this override to gpu_test instead? It shouldn't apply just to validation tests.

Reply:
I've moved it to gpu_test and handled case where they were no error scopes to pop. Let me know if that looks good to you.